src/app/common/confirmation-modal/confirm-dialog.service.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
```

Comment:
Please add space after `{` and before `}` to align with other files :) Please also check other files for this.